addons/rearm/XEH_postInit.sqf

```diff
@@ -1,11 +1,22 @@
 #include "script_component.hpp"
 
+// need these events before enabled check for zeus rearm
+[QGVAR(rearmEntireVehicleSuccessEH), LINKFUNC(rearmEntireVehicleSuccess)] call CBA_fnc_addEventHandler;
+[QGVAR(rearmEntireVehicleSuccessLocalEH), LINKFUNC(rearmEntireVehicleSuccessLocal)] call CBA_fnc_addEventHandler;
+
+if !GVAR(enabled) exitWith {};
+
 GVAR(hardpointGroupsCache) = [] call CBA_fnc_createNamespace;
 GVAR(configTypesAdded) = [];
 ["ace_settingsInitialized", {
     TRACE_2("settingsInit",GVAR(level),GVAR(supply));
-    ["LandVehicle", "Init", {_this call FUNC(initSupplyVehicle)}, true, ["StaticWeapon"], true] call CBA_fnc_addClassEventHandler;
+    ["AllVehicles", "Init", {_this call FUNC(initSupplyVehicle)}, true, ["Man", "StaticWeapon"], true] call CBA_fnc_addClassEventHandler;
     ["ReammoBox_F", "Init", {_this call FUNC(initSupplyVehicle)}, true, [], true] call CBA_fnc_addClassEventHandler;
+    ["House", "Init", {_this call FUNC(initSupplyVehicle)}, true, [], true] call CBA_fnc_addClassEventHandler;
+    // placed in editor static objects don't trigger init
+    {
+        _x call FUNC(initSupplyVehicle);
+    } forEach allMissionObjects "Static";
```

Comment:
How fast/slow is this?

Reply:
I think it's as slow as many of static objects placed in mission. I didn't test it on such mission but I don't think we have choice here.

Reply:
It's postInit. It doesn't matter if it's slow/ it is not slow enough to matter in postInit.

Reply:
`allMissionObjects` will hopefully also be optimized in 2.06 by @dedmen.